src/examples/java/io/nats/examples/autobench/JsPubBenchmark.java

```diff
@@ -26,7 +26,7 @@ public class JsPubBenchmark extends AutoBenchmark {
     private static final Map<String, String> SAVED_SUBJECTS = new HashMap<>();
 
     public static String getKey(long messageCount, long messageSize) {
-        return "" + messageCount + "x" + messageSize;
+        return messageCount + "x" + messageSize;
```

Comment:
Non production code who cares? Yeah it's not optimal and if you are working in this file for some actual code change or fix, great, change this, but otherwise no.